packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -632,6 +632,28 @@ fn.addEventSource(new eventsources.S3EventSource(bucket, {
 }));
 ```
 
+The following code adds an DynamoDB notification as an event source filtering insert events:
+
+```ts
+import * as eventsources from '@aws-cdk/aws-lambda-event-sources';
+import * as dynamodb from '@aws-cdk/aws-dynamodb';
+
+declare const fn: lambda.Function;
+const table = new dynamodb.Table(this, 'Table', {
+  partitionKey: {
+    name: 'id',
+    type: dynamodb.AttributeType.STRING,
+  },
+  stream: dynamodb.StreamViewType.NEW_IMAGE,
+});
+fn.addEventSource(new eventsources.DynamoEventSource(table, {
+  startingPosition: lambda.StartingPosition.LATEST,
+  filterCriteria: lambda.FilterCriteria.addFilters({
+    eventName: lambda.FilterRule.textEquals('INSERT'),
+  }),
```

Comment:
Thank you for you reply

Something like this would be better right?

```ts
  filters: [
    { eventName: lambda.FilterRule.textEquals('INSERT') },
  ]
```

Reply:
Yep, that's precisely what I was thinking.